Week3Project/Week3Project/Program.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+
+namespace Week3Project
+{
+    class Program : User
```

Comment:
This line could stay as `class Program` without the `: User`. We haven't gotten to inheritance or interfaces yet, but this is making the Program class inherit from the User class, which probably isn't needed